starter-code/index.js

```diff
@@ -1,11 +1,57 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor() {
+      this.items = [];
+      this.length = this.items.length;
+    }
+
+    add(item) {
+      this.items.push(item);
+      this.items.sort(function(a, b){
+        return a - b;
+      })
+      this.length += 1;
+    }
+
+    get(pos) {
+      if (pos >= 0 || pos <= this.items.length) {
+        return this.items[pos];
+      } else {
+        throw new Error("OutOfBounds");
+      }
+    }
+
+    max() {
+      if (this.items.length) {
+        return Math.max(...this.items);
+      } else {
+        throw new Error("EmptySortedList");
+      }
+    }
+
+    min() {
+      if (this.items.length) {
+        return Math.min(...this.items);
+      } else {
+        throw new Error("EmptySortedList");
+      }
+    }
+
+    average() {
```

Comment:
idem, il faut que tu indiques la condition de ton if : 
average() {
    if (this.length <= 0) {
      throw new Error("EmptySortedList");
    }
    return this.sum(this.items) / this.length;
  }